src/common/product_models/columnar_attribs_common_models.py

```diff
@@ -111,7 +111,7 @@ def _check_enums(self) -> Self:
     
     _RANGE_VALIDATORS = {
         ColumnType.INT: (lambda x: isinstance(x, int), "an integer"),
-        ColumnType.FLOAT: (lambda x: isinstance(x, float),"a float"),
+        ColumnType.FLOAT: (lambda x: isinstance(x, float) or isinstance(x, int), "a float"),
```

Comment:
I am not sure why we want to include `isinstance(x, int)` here. We did apply casting to float for columns of type ColumnType.FLOAT.

Reply:
👍